### DISCUSSION TITLE
Trying to use "folder as context" and result: empty (chat mode)

### DISCUSSION THREAD

**Discussion by Mara-Li** [logancyang/obsidian-copilot#303] [Q&A] [answered]
I want to use a folder as context:
1. I use the command `Set note context for chat mode` with: `10. PROJECT/Scientifique et médical/Endocrinologie`
2. I use the button to send context
3. Message:
```
> Please read the notes below and be ready to answer questions about them. 
	Chat context by path: 10. PROJECT/Scientifique et médical/Endocrinologie
> OK I've read these notes. Feel free to ask related questions, such as 'give me a summary of these notes in bullet points', 'what key questions does these notes answer', etc.
```
4. Trying to get any info: `give me a summary of these notes in bullet points`
5. `The note does not contain any information to summarize; it's empty.`

I have ~5 shorts note in this folder.

**Comment by Mara-Li** [comment]
It seems that it found no note in the folder: 
```
*** DEBUG INFO ***
user message: Please read the notes below and be ready to answer questions about them. If there's no information about a certain topic, just say the note does not mention it. The content of the note is between "/***/":

/***/

[]

/***/
```

**Comment by logancyang** [comment, marked as answer]
@Lisandra-dev You solved this yourself, congrats! Closing for now.
